### PR TITLE
Fix gan128

### PR DESCRIPTION
**Pull request by bryanlyon**
This is a fix to GAN128 to patch out the multi gpu until Keras updates their upstream issue (as mentioned in #287 )

In addition, while I was debugging, I fixed the exception handling in train.py so that it properly passes errors that it doesn't catch..

### CODE REVIEW

File: plugins/Model_GAN128/Model.py [hunk 1]
```diff
@@ -42,7 +44,7 @@ class GANModel():
 
     def __init__(self, model_dir, gpus):
         self.model_dir = model_dir
-        self.gpus = gpus
+        self.gpus = 1 #See https://github.com/deepfakes/faceswap/issues/287
```
Comment (Clorr) [inline plugins/Model_GAN128/Model.py]: I'd prefer you let the original line and put below it an assert, with a failure message pointing to the issue you mention in the comment

File: scripts/train.py [hunk 2]
```diff
@@ -178,7 +178,7 @@ def processThread(self):
                 print('Saving model weights has been cancelled!')
             exit(0)
         except Exception as e:
-            print(e)
+            raise e
```
Comment (Clorr) [inline scripts/train.py]: Somehow I agree that the print(e) is not convenient as it make a more readable error message but that masks the details. However I would be prefere this to be made in another PR

**Comment by bryanlyon** [comment]
Sure, I'll send another PR and remove it from this one.

**Comment by bryanlyon** [comment]
I'll do an assert instead of just doing single.  I'll update this soon.

**Comment by bryanlyon** [comment]
I've made the changes as requested.  If there are not other concerns, this is ready to merge.
